index.html

```diff
@@ -26,49 +26,289 @@
-          </li>
-          <li>
-            <a href="#"
-              ><img src="/assests/images/Linkedin icon.png" alt=""
-            /></a>
-          </li>
-        </ul>
+      <div class="myWork myWork4">
+        <div class="snapshot-port">
+          <img
+            src="/assests/images/Snapshoot Portfolio (3).png"
+            alt=""
+            class="multi-post-img"
+          />
+        </div>
+        <div class="left-block left-block-multi">
+          <div class="primary-text">
+            <h1>Multi-Post Stories</h1>
+          </div>
+
+          <ul class="second-list">
+            <li class="canopy">CANOPY</li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; Back End
+              Dev
+            </li>
+            <li class="other-li">
+              <img src="/assests/images/Counter.png" alt="" /> &nbsp; 2015
+            </li>
+          </ul>
+          <div class="work-statment">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="see-proj">
+            <a href="" class="lang">html</a>
+            <a href="" class="lang">css</a>
+            <a href="" class="lang js-anc">javaScript</a>
+            <button class="buttn">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="section-3">
+      <div class="container">
+        <div class="about-me-2">
+          <div class="headline">
+            <h1>
+              About <br />
+              Myelf
+            </h1>
+          </div>
+          <div class="about-me">
+            <p class="abt-me">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <a class="meet">LET'S CONNECT</a>
+          <div class="social-icons">
+            <ul class="social-icons-list">
+              <li>
+                <a href="#"><img src="assests/images/Vector.png" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (1).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (2).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assests/images/Vector (3).png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="/assests/images/Linkedin icon.png" alt=""
+                /></a>
+              </li>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image